.circleci/config.yml

```diff
@@ -683,6 +685,9 @@ jobs:
       parallel_num:
         type: integer
         default: 3
+      test_suites:
+        type: string
+        default: '".*"'
```

Comment:
Using option `-g` to run all tests or just targeted test suites. 
`-g, --grep <grep>    Only run tests matching this regular expression (default: ".*")`